connectors/src/connectors/zendesk/lib/sync_ticket.ts

```diff
@@ -232,7 +223,11 @@ ${comments
         `title:${ticket.subject}`,
         `updatedAt:${updatedAtDate.getTime()}`,
         `createdAt:${createdAtDate.getTime()}`,
-        ...filterCustomTags(ticket.tags, logger),
+        ...metadataAsTags,
+        ...filterCustomTags(
```

Comment:
As discussed IRL let's align on Intercom and put the custom_fields as system tags?